.circleci/testnet-remove-chain.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+/*
+This script updates a local Boyar config.json file to remove an existing PR chain
+
+Usage: 
+
+$ testnet-remove-chain.js <ChainID>
+
+Examples:
+
+$ testnet-remove-chain.js 1104832
+
+*/
+
+const path = require('path');
+const fs = require('fs');
+
+const targetChainId = process.argv[2];
+const configFilePath = path.join(process.cwd(), 'config.json');
+
+if (!targetChainId) {
+    console.log('No chainId given!');
+    process.exit(1);
+}
+
+const configuration = require(configFilePath);
+
+const chainIndex = configuration.chains.findIndex(chain => chain.Id === parseInt(targetChainId));
```

Comment:
You do this manipulation quite a lot, maybe it should be extracted to a separate file.

Reply:
I agree, I'll refactor the common logic to one file and use it where needed.